netlink-packet-route/src/rtnl/link/nlas/mod.rs

```diff
@@ -500,7 +501,15 @@ impl<'a, T: AsRef<[u8]> + ?Sized> ParseableParametrized<NlaBuffer<&'a T>, u16> f
                     }
                     AfSpecInet(nlas)
                 }
-                AF_BRIDGE => AfSpecBridge(payload.to_vec()),
+                AF_BRIDGE => {
+                    let mut nlas = vec![];
+                    let err = "invalid IFLA_AF_SPEC value";
```

Comment:
This should be `"invalid AF_BRIDGE value"`.

Reply:
Fixed.